tests/test_future/test_int.py

```diff
@@ -265,6 +265,9 @@ def test_small_ints(self):
     def test_no_args(self):
         self.assertEqual(int(), 0)
 
+    @unittest.skipIf(sys.version_info >= (3, 7),
```

Comment:
The test for `int('100', base=2)` shouldn't be skipped.

Reply:
Yeah, this one still works but I didn't think it was worth keeping. Would you like me to split the test in two functions.

Reply:
Yes please! :)